conda_forge_tick/migrators/round_trip.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from conda_forge_tick.migrators.core import MiniMigrator, skip_migrator_due_to_schema
+from conda_forge_tick.recipe_parser import CondaMetaYAML, get_yaml_parser
+
+
+class YAMLRoundTrip(MiniMigrator):
+    allowed_schema_versions = (0, 1)
+
+    def filter(self, attrs, not_bad_str_start=""):
+        return skip_migrator_due_to_schema(attrs, self.allowed_schema_versions)
+
+    def migrate(self, recipe_dir, attrs, **kwargs):
+        fname = os.path.join(recipe_dir, "meta.yaml")
+        if os.path.exists(fname):
+            try:
+                with open(fname) as f:
+                    meta = CondaMetaYAML(f.read())
+                with open(fname, "w") as f:
+                    meta.dump(f)
+            except Exception:
+                pass
```

Comment:
why do we have this broad exception clause here?

Reply:
The minimigrators can never fail. Sometimes recipes are unparsable due to unsupported syntax, duplicate yaml keys, etc. So we swallow all errors.